netbox_branching/templates/netbox_branching/inc/share_button.html

```diff
@@ -0,0 +1,6 @@
+{% if active_branch %}
+    {% load i18n %}
+    <a href="{{ url }}?_branch={{ active_branch.schema_id }}" type="button" class="btn btn-outline-blue">
```

Comment:
We should add a (translated) title text to the link explaining its purpose (e.g. "Link to this object in the current branch"). This could potentially replace the button's text to save space.